src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -215,6 +215,7 @@ class CrossDeviceLinkUI extends Component {
                 className={classNames(style.btn, {[style.sending]: this.state.sending})}
                 variants={["primary"]}
                 onClick={this.sendSms}
+                disabled={this.state.sending}
```

Comment:
I don't think this is a safe way of solving the issue because the value of react state might be asynchronous.  I think we need to prevent the request on line 173 from firing if it has been fired already within n milliseconds

Reply:
So would you say disabling the button and preventing the request itself happening at line 173 will be the best solution? Or just sticking to preventing the request firing again within the n miliseconds of being initially fired?

Reply:
Yes, I think the best way would be preventing the request from firing again within n milliseconds of being initially fired. @rfreitas thoughts on this?

Reply:
Actually, maybe an onClick handler would be better. Where the sending state is set in the handler only if it is not already sending.

Reply:
Yes. Btw I can see that the bug is still present so this change is not having the desired effect.

Reply:
As discussed, looks like the latest code change(s) has not been picked up since the build is failing :( 
Will need to get that sorted out first.

Reply:
In the meantime have committed additional changes made to make the click handling more robust